docs/pages/connect-your-client/gui-clients.mdx

```diff
@@ -71,34 +71,16 @@ Ensure that your environment includes the following:
 
```

Comment:
I know this wasn't part of the PR, but at the top of the "Get connection information" section, I'm not sure a reader will be able to determine which `TabItem` to select. Can we include a quick introduction to what "Authenticated Proxy", "TLS Routing", and "Separate Ports" mean, and when a reader should select one of these tabs?

Reply:
@stevenGravy What do you think about the comment above?